test/fail_compilation/fail4611.d

```diff
@@ -1,16 +0,0 @@
-/*
-TEST_OUTPUT:
----
-fail_compilation/fail4611.d(15): Error: Vec[1000000000] size 4 * 1000000000 exceeds 16MiB size limit for static array
----
-*/
-
-struct Vec
-{
-    int x;
-}
-
-void main()
-{
-    Vec[1000_000_000] a;
-}
```

Comment:
@NightmareX1337 I think the easiest is to use a shell script. The shell script will compile and run the actual test. Add a guard that ensures the test is only run on Windows. This can be done by exiting the script before doing anything for all non Windows platforms. See this existing test for an example [1].

[1] https://github.com/dlang/dmd/blob/master/test/runnable/test16096.sh